lib/matplotlib/axes/_base.py

```diff
@@ -2481,7 +2481,7 @@ def relim(self, visible_only=False):
 
     def update_datalim(self, xys, updatex=True, updatey=True):
         """
-        Extend the `~.Axes.dataLim` Bbox to include the given points.
+        Extend the ``ax.dataLim`` Bbox to include the given points.
```

Comment:
Ah, this is documented as an attribute to the class https://matplotlib.org/devdocs/api/axes_api.html#matplotlib.axes.Axes, but apparently one cannot link to these definition :(